docs/cli/cozy-stack_config_insert-asset.md

```diff
@@ -6,14 +6,11 @@ Inserts an asset
 
 Inserts a custom asset in a specific context
 
-```
-cozy-stack config insert-asset --url <url> --name <name> --shasum <shasum> --context <context> [flags]
-```
+Deprecated: please use the command cozy-stack assets add.
```

Comment:
Can we add a link to `cozy-stack_assets_add` ?

Reply:
Well, it is a bit complicated as those pages are generated by the same thing that shows help in terminal.